src/lib/renderComponent.tsx

```diff
@@ -109,16 +111,20 @@ const renderComponent = <P extends {}>(
 
   return (
     <FelaTheme
-      render={({
-        siteVariables = {},
-        componentVariables = {},
-        componentStyles = {},
-        rtl = false,
-        renderer = felaRenderer,
-      }: IThemeInput | IThemePrepared = {}) => {
+      render={(theme: IThemePrepared) => {
+        const {
+          siteVariables = {},
+          componentVariables = {},
+          componentStyles = {},
+          icons = {},
+          rtl = false,
+          renderer = felaRenderer,
+        } = theme
+
         const ElementType = getElementType({ defaultProps }, props)
 
-        const stateAndProps = { ...state, ...props }
+        const derivedProps = getExtraProps({ icons })
```

Comment:
It looks like you took old changes from #183.
`getExtraProps` was removed.

Reply:
thanks for the heads up - this artefact had leaked in during merge, as there were lots of conflicts with master. Thanks a lot!